### PR TITLE
Debermudez port existing tests

### PR DESCRIPTION
**Pull request by debermudez**

### CODE REVIEW

File: qa/L0_nvtx/test.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+apt-get update && \
+    apt-get install -y --no-install-recommends python
+
+rm -f $TEST_LOG
+RET=0
+
+set +e
+
+(cd /opt/pytorch/pyprof/qa/L0_nvtx/ && \
```
Comment (dzier) [inline qa/L0_nvtx/test.sh:27]: 1. The CI is already in this directory. That is how it is running the test.sh file in the first place, so remove this `cd`. 
2. Remove the set +e above so we can see that this script is running. Right now it doesn't look like anything is reunning

File: qa/L0_pyprof_data/test.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+apt-get update && \
+    apt-get install -y --no-install-recommends python
+
+rm -f $TEST_LOG
+RET=0
+
+set +e
+
+(cd /opt/pytorch/pyprof/qa/L0_pyprof_data/ && \
```
Comment (dzier) [inline qa/L0_pyprof_data/test.sh:27]: Same comment as above.

File: qa/L0_pyprof_data/test.sh [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+
+apt-get update && \
+    apt-get install -y --no-install-recommends python
+
+rm -f $TEST_LOG
+RET=0
+
+./test_pyprof_data.py > $TEST_LOG 2>&1
+if [ $? -ne 0 ]; then
```
Comment (dzier) [inline qa/L0_pyprof_data/test.sh:26]: Does this output any files? If so, the should be a check to see if the files exist and not zero.

Comment (debermudez) [reply qa/L0_pyprof_data/test.sh:26]: The file output is just a cat of the test output. 
I thought it was standard practice to do that in our testing. I don't rely on the presence of the log file to pass. I just want the output of the run in the CI if it fails to aid in debugging.

**Pull request merged**